src/erlastic_search.erl

```diff
@@ -648,6 +652,34 @@ aliases(Body) ->
 aliases(Params, Body) ->
     erls_resource:post(Params, filename:join([<<"_aliases">>]), [], [], erls_json:encode(Body), Params#erls_params.http_client_options).
 
+%%--------------------------------------------------------------------
+%% @doc
+%% Takes a list of operations of the type {Operation, Index, Type, Id, Json}
+%% Performs these operations using _bulk endpoint.
+%% The possible operations are index, create, delete and update.
+%% (see the doc at https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html)
+%% @end
+%%--------------------------------------------------------------------
+-spec bulk_operation(operations()) -> {ok, list()} | {error, any()}.
+bulk_operation(OperationIndexTypeIdJsonTuples) ->
+    bulk_operation(#erls_params{}, OperationIndexTypeIdJsonTuples).
+
+-spec bulk_operation(#erls_params{}, operations()) -> {ok, list()} | {error, any()}.
+bulk_operation(Params, OperationIndexTypeIdJsonTuples) ->
+    Body = lists:map(fun
+                       Build({delete, Index, Type, Id}) ->
+                         Build({delete, Index, Type, Id, [], []});
+                       Build({delete, Index, Type, Id, HeaderInformation}) ->
+                         Build({delete, Index, Type, Id, HeaderInformation, []});
```

Comment:
Same comment as last review. I believe that's not generating a correct ES request (adding an empty JSON list after `delete` lines)